test/testutil/init_actor.hpp

```diff
@@ -21,13 +21,22 @@ std::shared_ptr<fc::vm::state::StateTree> setupInitActor(
         std::make_shared<fc::storage::ipfs::InMemoryDatastore>());
   }
   auto store = state_tree->getStore();
-  fc::vm::actor::builtin::v0::init::InitActorState init_state;
-  init_state.address_map_0 = {store};
+  using namespace fc;
```

Comment:
Using `using namespace` in .h or .hpp is not good.
Please put this code in the necessary namespace

Reply:
`using namespace` is discouraged in `namespace` level, this line affects only this function

Reply:
Ok